### ISSUE TITLE
Module not found: Package path ./style.css is not exported from package

### ISSUE THREAD

**Issue by amkuchta** [nosferatu500/react-sortable-tree#9]
# Reporting a Bug?
Using your repo with NextJS / Typescript, I'm getting the following error:

```
./pages/_app.tsx:6:0
Module not found: Package path ./style.css is not exported from package /Users/alexkuchta/Documents/Code/rmfer/front-end/node_modules/@nosferatu500/react-sortable-tree (see exports field in /Users/alexkuchta/Documents/Code/rmfer/front-end/node_modules/@nosferatu500/react-sortable-tree/package.json)
  4 | import '../styles/fontawesome.css';
  5 | import "react-loader-spinner/dist/loader/css/react-spinner-loader.css";
> 6 | import '@nosferatu500/react-sortable-tree/style.css'
  7 | 
  8 | function MyApp({ Component, pageProps }: AppProps) {
  9 |   return <Component {...pageProps} />;

Import trace for requested module:

https://nextjs.org/docs/messages/module-not-found
```

**Comment by nosferatu500** [comment]
Hi! Thank you for reporting this.

style.css is also exported with other files (just checked in my side project) but I'm not sure if it works with NextJS, I'll check it later.

**Comment by reloadedhead** [comment]
I was about to post this, happened to me with React/TS as well. Exporting it via `exports` in `package.json` seems to do the job:
```
"exports": {
    "./package.json": "./package.json",
    ".": {
      "types": "./index.d.ts",
      "module": "./esm/index.js",
      "import": "./esm/index.mjs",
      "default": "./index.js"
    },
  "./style.css": "./style.css"
  },
```

**Comment by amkuchta** [comment]
@reloadedhead @nosferatu500 the above fix did not work for me - it resulted in a `webpackMissingModule` error:

```
Error: Cannot find module '@nosferatu500/react-sortable-tree/style.css' at webpackMissingModule (webpack-internal:///./pages/_app.tsx:13:50) at eval (webpack-internal:///./pages/_app.tsx:13:168) at Object../pages/_app.tsx (/Users/amkuchta/Documents/Code/rmfer/front-end/.next/server/pages/_app.js:22:1) at __webpack_require__ (/Users/amkuchta/Documents/Code/rmfer/front-end/.next/server/webpack-runtime.js:33:42) at __webpack_exec__ (/Users/amkuchta/Documents/Code/rmfer/front-end/.next/server/pages/_app.js:84:39) at /Users/amkuchta/Documents/Code/rmfer/front-end/.next/server/pages/_app.js:85:28 at Object.<anonymous> (/Users/amkuchta/Documents/Code/rmfer/front-end/.next/server/pages/_app.js:88:3) at Module._compile (node:internal/modules/cjs/loader:1101:14) at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10) at Module.load (node:internal/modules/cjs/loader:981:32) {code: "MODULE_NOT_FOUND", stack: "Error: Cannot find module '@nosferatu500/react-sor…le.load (node:internal/modules/cjs/loader:981:32)", message: "Cannot find module '@nosferatu500/react-sortable-tree/style.css'"}
```

**Comment by amkuchta** [comment]
MY "quick and dirty" workaround was to copy the contents of the file to a local CSS, then import that.

**Comment by nosferatu500** [comment]
@amkuchta @reloadedhead 

The issue was fixed in build v4.0.2

Thanks for reporting!

**Comment by amkuchta** [comment]
@nosferatu500 thank you for the quick turn on this!
